sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/HttpConstants.java

```diff
@@ -254,6 +254,9 @@
         public static final String IS_BATCH_ORDERED = "x-ms-cosmos-batch-ordered";
         public static final String IS_BATCH_REQUEST = "x-ms-cosmos-is-batch-request";
         public static final String SHOULD_BATCH_CONTINUE_ON_ERROR = "x-ms-cosmos-batch-continue-on-error";
+
+        //Client telemetry header
```

Comment:
nit: please change to be of consistent formatting with other comments in this file.
```suggestion
        // Client telemetry header
```